Automattic/jetpack
# Quality: Update WorDBless

- Update to use latest WorDBless install scheme
- Changelogs





The WorDBless package had to update due to upstream changes with roots/wordpress. The new version required a different installer command.

#### Changes proposed in this Pull Request:

* Updates all projects using WorDBless to the new install scheme.
* Bump the versions for WorDBless that were not tracking dev-master.
* Update the transport-helper project to correctly install WorDBless and the corresponding gitignore update.

#### Other information:


#### Jetpack product discussion


p1663100254634889-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On a fresh copy of the repo (or at least in a particular project using WorDBless, that is no /vendor , /wordpress/ dirs), run jetpack install [--all|<project>]
* The project using WorDBless should have a wordpress dir with a wp-content/db.php file included.
* Before this patch, there would be an error about being unable to copy the db.php file.
